.changeset/honest-foxes-scream.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-splunk-on-call': minor
```

Comment:
@jhaals I'm happy to go with what you recommend. However, [semver.org](https://semver.org/) stipulates...

> MAJOR version when you make incompatible API changes,
> MINOR version when you add functionality in a backwards compatible manner, and
> PATCH version when you make backwards compatible bug fixes.

...and, in my mind, this _adds_ new functionality, but does so in a non-breaking, backwards compatible manner, hence my decision to specify `minor`. Does that change your view or do you still recommend I denote this as a `patch` change?

Reply:
Whilst packages are also below `<1.0.0` breaking changes become `minor` and `minor` becomes `patch` purely so that when you're buildling on top of it and making a load of breaking changes, when you're ready to go stable, you don't find yourself on version `55.0.0` as the first stable :)